qa/suites/rados/singleton-nomsgr/all/large-omap-object-warnings.yaml

```diff
@@ -0,0 +1,22 @@
+roles:
+- [mon.a, mgr.x, osd.0, osd.1, client.0]
+overrides:
+  ceph:
+    log-whitelist:
+      - (OSDMAP_FLAGS)
```

Comment:
nit: should escape parens in these whitelists with \ - they're passed to the egrep command